qcengine/programs/tests/test_pylibefp.py

```diff
@@ -0,0 +1,274 @@
+import copy
+import pytest
+
+from qcelemental.testing import compare, compare_recursive, compare_values
+
+import qcengine as qcng
+from qcengine.testing import using_pylibefp
+
+b2a = 0.529177
```

Comment:
I assume efp uses its own conversion? Can we add a comment here about that.

Reply:
I'll look it up, but this particular one is just to get the geometry in with both round numbers and to match the answers taken from libefp test suite. outputs are all in Eh, a0. So this conversion isn't a broad concern.

Reply:
Makes sense. Should be fine as long as we note it.